classes/02class/exercises/c02-network03/christina-roperto/c02-network023.txt

```diff
@@ -0,0 +1,25 @@
+# VPC
+Create a new AWS VPC with:
+
+CIDR block /16
+Name: devopsacademy-vpc
+No IPv6
+Tenancy default
+Post details from the vpc created.
+
+-----------------------------------------------
+Subnet ID : subnet-0d1bdd61c393a4a06
+VPC : vpc-06763d321855b5d7b | devopsacademy-vpc
+IPv4 CIDR : 10.0.0.0/24
```

Comment:
just pay attention for the subnets on the next exercise as you will only have a /24 to slice into smaller subnets -- good work on picking a smaller range!